app/models/pair_programming_exercise_feedback.rb

```diff
@@ -12,6 +12,32 @@ class PairProgrammingExerciseFeedback < ApplicationRecord
   scope :intermediate, -> { where.not(normalized_score: 1.00) }
   scope :final, -> { where(normalized_score: 1.00) }
 
+  enum comment: {
+    difficulty_easy: 0,
```

Comment:
Should the link to the answer options in the locals remain or should the texts also be dragged into the model?

Reply:
I would propose to remove them from the locales as well, so that we _just_ have the model and the database table.

When doing so, I would recommend adding a comment to the enum if the label chosen is not clear enough. Then, a future researcher can still gather meaningful insights from the data.